src/modules/fundingManager/oracle/interfaces/IFM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -0,0 +1,185 @@
+        uint fee_, uint maxFee_
+    );
+
+    /// @notice	Thrown when the oracle contract does not implement the required
+    ///         interface.
+    error Module__FM_PC_ExternalPrice_Redeeming_InvalidOracleInterface();
+
+    /// @notice	Thrown when third-party operations are disabled.
+    error Module__FM_PC_ExternalPrice_Redeeming_ThirdPartyOperationsDisabled();
+
+    /// @notice	Thrown when a redemption queue execution fails.
+    error Module__FM_PC_ExternalPrice_Redeeming_QueueExecutionFailed();
+
+    /// @notice Thrown when the project treasury address is invalid.
+    error Module__FM_PC_ExternalPrice_Redeeming_InvalidProjectTreasury();
+
+    // -------------------------------------------------------------------------
+    // Events
+
+    /// @notice	Emitted when reserve tokens are deposited.
+    /// @param	depositor_ The address depositing tokens.
+    /// @param	amount_ The amount deposited.
+    event ReserveDeposited(address indexed depositor_, uint amount_);
+
+    /// @notice	Emitted when a new redemption order is created.
+    /// @param	orderId_ Order identifier.
+    /// @param	seller_ Address selling tokens.
+    /// @param	receiver_ Address who receives the redeemed tokens.
+    /// @param	sellAmount_ Amount of tokens to sell.
+    /// @param	exchangeRate_ Current exchange rate.
+    /// @param	collateralAmount_ Amount of collateral.
+    /// @param	feePercentage_ Fee percentage applied.
+    /// @param	feeAmount_ Fee amount calculated.
+    /// @param	redemptionAmount_ Final redemption amount.
+    /// @param	collateralToken_ Address of collateral token.
+    /// @param	redemptionTime_ Time of redemption.
+    /// @param	state_ Initial state of the order.
+    event RedemptionOrderCreated(
+        uint indexed orderId_,
+        address indexed seller_,
+        address indexed receiver_,
+        uint sellAmount_,
+        uint exchangeRate_,
+        uint collateralAmount_,
+        uint feePercentage_,
+        uint feeAmount_,
+        uint redemptionAmount_,
+        address collateralToken_,
+        uint redemptionTime_,
+        RedemptionState state_
+    );
+
+    // -------------------------------------------------------------------------
+    // View Functions
+
+    /// @notice	Gets the current open collateral redemption amount.
+    /// @return	amount_ The total amount of open redemptions.
+    function getOpenRedemptionAmount() external view returns (uint amount_);
+
+    /// @notice	Gets the next available order ID.
+    /// @return	orderId_ The next order ID.
+    function getNextOrderId() external view returns (uint orderId_);
+
+    /// @notice	Gets the current order ID.
+    /// @return	orderId_ The current order ID.
+    function getOrderId() external view returns (uint orderId_);
+
+    /// @notice Gets the project treasury address.
+    /// @return address_ The address of the project treasury.
+    function getProjectTreasury() external view returns (address);
```

Comment:
Same as in the contract, unnamed return value.

Reply:
added